src/gengowatcher/config.py

```
@@ -163,12 +186,42 @@
+                                config_modified = True
+                                normalized_paths = True
+                except Exception:
+                    pass
+
                 # Save config if it was modified
                 if config_modified:
                     try:
                         with open(self.CONFIG_FILE, "w", encoding="utf-8") as f:
                             self._config_parser.write(f)
-                        print(f"Config file updated with missing sections/options")
+                        if added_defaults and normalized_paths:
```

Ruff (0.13.1)

Undefined name `added_defaults`

(F821)